lms/product/canvas/_plugin/misc.py

```diff
@@ -161,6 +161,17 @@ def get_deeplinking_launch_url(self, request, assignment_configuration: dict):
             .geturl()
         )
 
+    def is_speed_grader_launch(self, request) -> bool:
```

Comment:
Moved this to a common location. This plugin is the best place for it even it is not the perfect fit.